paths.tex

```diff
@@ -169,11 +166,10 @@ \section{Consecutive Contemporaneity}
 if they did not happen at the same time. This is formally defined below.
 
 \begin{defn}
-  A \textbf{consecutive temporal path} between $v_1$ and $v_n$ is a path
-  $P = v_1, v_2, \cdots, v_n$ such that $v_i \in V$ for all $i \in [n]$,
-  and $v_jv_{j+1} \in E$, for all $j \in [n-1]$, and for every pair of
-  consectutive edges $(v_{i-1},v_{i})_{t_1}$ and $(v_{i}, v_{i+1})_{t_2}$,
-  $t_1 \leq t_2$.
+  \label{defn:cons_temp_path}
+  A \textbf{consecutive temporal path} between $u$ and $v$ is a $v_1,v_n$-path
+  $P = e_1, e_2, \cdots, e_n$ such that for consecutive edges $e_i,e_{i+1} \in P$,
+  $T(e_i) \cap T(e_j) \neq \emptyset$.
```

Comment:
Should we add in the generalize minimum shape graphs for clarity? Or at the very least some of the graphs we were drawing on the board -- they are an easy way to see the subtle differences in the types of paths we are defining.